cpp/CMakeLists.txt

```diff
@@ -556,7 +556,6 @@ add_library(cudf
             src/copying/copy_range.cu
             src/filling/legacy/fill.cu
             src/filling/fill.cu
-            src/filling/legacy/repeat.cu
```

Comment:
We shouldn't be removing legacy APIs yet as cuSpatial hasn't ported over yet.

Reply:
sgtm. I'll revert.